apps/src/code-studio/components/ModelManagerDialog.jsx

```diff
@@ -81,7 +81,7 @@ export default class ModelManagerDialog extends React.Component {
           handleClose={this.closeModelManager}
           useUpdatedStyles
         >
-          <h2>A.I Trained Models</h2>
+          <h2>AI Trained Models</h2>
```

Comment:
Thanks for following up on this detail! Can you add this string to the i18n pipeline, please? You can do that by adding a new key to https://github.com/code-dot-org/code-dot-org/blob/staging/apps/i18n/common/en_us.json and set the value to the English string "AI Trained Models".

Reply:
oops - sorry for the double comment - looks like Anna and I are on the same page 😄

Reply:
Thank you both for looking at this! I added it in :)